applications/asset_tracker/src/main.c

```diff
@@ -293,6 +297,7 @@ static void sensor_trigger_handler(struct device *dev,
 	flip_send(NULL);
 }
 
+#if defined(CONFIG_DK_LIBRARY)
```

Comment:
This works, but using `CONFIG_DK_LIBRARY` directly to toggle all the UI stuff in main.c may create some headaches for someone porting the app to a board not supported by that library down the line (i.e. a custom board). Should we make a new configuration value like `CONFIG_USE_UI_MODULE` to avoid that? That would contain the dependency in ui.c for now.